tests/vcd_catalog.py

```diff
@@ -22,5 +22,14 @@ def test_catalog_control_access_retrieval(self):
         control_access = org.get_catalog_access_control_settings(catalog.get('name'))
         assert control_access is not None
 
+    def test_change_catalog_owner(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
```

Comment:
I was under the impression that org-admin and sys-admin can only change the owner of a catalog. If that is the case, should the Org object be created with is_admin=True?